client/css/reviewRating.scss

```diff
@@ -121,10 +126,34 @@
   margin: 5px auto;
 
 }
+.input_label span{
+  width: 20%;
+}
+.radios{
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+}
+.radios div{
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.radios div span{
+  width: 20%;
+}
+
+#word_count{
+  width: 80%;
+  font-size: x-small;
```

Comment:
Can we give this an actual size here rather than `x-small`. I'm sure it works and it looks good, but no one but you knows what x-small is in this context. Start thinking about the people that will be maintaining and extending your code down the road. How can you make their job easier?